journal/updated/09-chapter-8.adoc

```diff
@@ -195,50 +199,50 @@ and by me rather desirable that it may soon be experienced.
 
 On the 17th of first month, 1785, we left Bristol,
 having the blessed reward of pure peace for our labours.
-Lydia Hawksworth accompanied us to Frenchay, where we had a meeting;
-several Friends from the city were there, and we were owned by our great Lord,
-his name was magnified, and his own works did praise him.
+Lydia Hawksworth accompanied us to Frenchay, where we had a meeting.
+Several Friends from the city were there, and we were acknowledged by our great Lord;
+His name was magnified, and His own works did praise Him.
 
-Early on the following morning, accompanied by Thomas Scantlebury, we went to Painswick,
-and were at their meeting on the 19th; and at Tewkesbury on the 21st.
+Early the following morning, accompanied by Thomas Scantlebury, we went to Painswick,
+and were at their meeting on the 19th, and at Tewkesbury on the 21st.
 
-On seventh-day we reached Worcester, and lodged at our kind friend Timothy Bevington`'s:
-were at their meetings on first-day, where were also Patience Brayton and her companion.
+On seventh-day we reached Worcester, and lodged at our kind friend Timothy Bevington`'s.
+Were at their meetings on first-day, where were Patience Brayton and her companion also.
 On second-day, the 24th, my dear companion left me, she being very desirous to get home;
-but I was not easy to leave Worcester so soon, and stayed more than a week longer,
+but I was not clear to leave Worcester so soon, and stayed more than a week longer,
```

Comment:
You're going to find this a lot in the journals from this time period. "I was not easy to..." and I think you can leave them. "Clear" isn't bad. But easy is good too.